slp_drawio/slp_drawio/load/drawio_mapping_file_loader.py

```diff
@@ -0,0 +1,53 @@
+from slp_base import MappingLoader, LoadingMappingFileError
+from deepmerge import always_merger
+import yaml
+import logging
+import jmespath
+
+logger = logging.getLogger(__name__)
+
+
+class DrawioMapping:
+
+    def __init__(self, trustzones, components, dataflows):
+        self.mapping_trustzones = trustzones
+        self.mapping_components = components
+        self.mapping_dataflows = dataflows
+
+
+class DrawioMappingFileLoader(MappingLoader):
+
+    def __init__(self, mapping_data_list: [bytes]):
+        self.provided_mappings = mapping_data_list
+        self.merged_mappings = {}
+        self.drawio_mapping = None
+
+    def load(self):
+        try:
+            self.__merge_mapping()
+            self.drawio_mapping = DrawioMapping(
+                self.__get_trustzones(),
+                self.__get_components(),
+                []
+            )
+        except Exception as e:
+            raise LoadingMappingFileError('Error loading the mapping file. The mapping files are not valid.',
+                                          e.__class__.__name__, str(e))
+
+    def __get_trustzones(self):
+        trustzone_mappings_list = jmespath.search("trustzones", self.merged_mappings)
+        return dict(zip([tz['label'] for tz in trustzone_mappings_list], trustzone_mappings_list))
+
+    def __get_components(self):
+        component_mappings_list = jmespath.search("components", self.merged_mappings)
+        return dict(zip([tz['label'] for tz in component_mappings_list], component_mappings_list))
```

Comment:
This logic (equals to Visio in MAIN branch) has been changed in DEV as it is not suitable for all the UC.
Review https://github.com/iriusrisk/startleft/blob/319ed1628d09975ac3d5d6ae535476f79f6d1e91/slp_visio/slp_visio/load/visio_mapping_loader.py#L41 to check if it is better for this case